dts/arm/nuvoton/npcx7m6fb.dtsi

```diff
@@ -453,12 +453,6 @@
 			interrupt-names = "kbc_ibf", "kbc_obe", "pmch_ibf",
 					  "pmch_obe", "p80_fifo";
 
-			/* Host serial port pinmux PIN75 86 36 33 42 C7 B3 B2 */
-			pinctrl = <&altb_rxd_sl &altb_txd_sl
```

Comment:
Are these these the pins needed for LPC/eSPI communication? If so can we leave them here in the and make the status of this device "disabled" instead?

Would it be possible to enable the LPC device and leave the io_host_uart disabled? When would that scenario exist?

Reply:
> Are these these the pins needed for LPC/eSPI communication? If so can we leave them here in the and make the status of this device "disabled" instead?
[ML] These pins are independent of LPC/eSPI communication. eSPI works normally if pin-mux of host uart is off.
> Would it be possible to enable the LPC device and leave the io_host_uart disabled? When would that scenario exist?
[ML] As far as I know, there is only one status property in one device with the same compatible. 

The following is my original plan for pin-mux. Any comment is appreciated.
First, the "status" of internal modules without io-pads such as pcc, scfg, host_sub, and so on are "okay" by default. You need to turn them on by enabling CONFIG_CLOCK_CONTROL, CONFIG_PINMUX, and CONFIG_ESPI in defconfig or prj files.

Then, the "status" of devices with io-pads (with "pinctrl" property) besides GPIO must be "disabled" by default. Besides turning their configuration, you also need to set their status to "okay" in dts file explicitly. For example, the following steps are necessary if we need host uart functionality in our application.

1. Add CONFIG_ESPI=y and CONFIG_ESPI_PERIPHERAL_UART=y in prj file of samples folder.
2. Add &host_uart{ status = "okay";}; in dts file.

The benefit of this method is easy to observe the pin-mux usages of all io-pads in npcx ec in dts file. By this simple rule, we needn't to spend time to check the actual pin-mux usage in our driver.

Reply:
Sounds good. Thanks for the explanation.